src/jitlayers.cpp

```diff
@@ -1324,3 +1333,8 @@ size_t jl_jit_total_bytes_impl(void)
 {
     return jl_ExecutionEngine->getTotalBytes();
 }
+
+extern "C" JL_DLLEXPORT
+LLVMContextRef jl_get_ee_context_impl(void) {
+    return wrap(jl_ExecutionEngine->getContext().getContext());
```

Comment:
So technically this is wrong, since you return the underlying context without first locking the TSCtx.

Reply:
I've added this as a small hack to get `@code_llvm` working since creating a new context is still causing some issues in codegen that I haven't tracked down yet; I intend to remove this method and thus the locking issue as soon as multiple contexts can get through codegen without issue. Since right now the context isn't used except under the codegen lock, we should temporarily be safe until that other issue is resolved.